pymc4/_hmc/nuts.py

```diff
@@ -5,7 +5,8 @@
 import numpy as np
 import numpy.random as nr
 
-from ..arraystep import Competence
+# Array step doesn't seem to exist
```

Comment:
```suggestion
# FIXME Array step doesn't seem to exist
```

Reply:
I removed these based on Thomas's reply to here
https://github.com/pymc-devs/pymc4/issues/50